.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
+exclude = **/*.sikuli/*,**/*_env/*,**/node_modules/*,**/venv/**
```

Comment:
We don't care about any auto-generated Sikuli Python scripts, or any Python files that may be in a venv (or even random Python scripts which may be bundled with a Node.js module).